fuzz-tests/ntt.cpp

```diff
@@ -21,49 +21,52 @@ ll modpow(ll a, ll e) {
 }
 
 typedef vector<ll> vl;
```

Comment:
Ooops, I started commenting on the fuzz test rather than the original code. Is it possible to use an `#include` here? Perhaps with some trickery for getting the right `mod`, e.g. `namespace ignored { #include "modpow.h" } #include "ntt.h"` or `#define`s